packages/remix-server-runtime/routeModules.ts

```diff
@@ -1,5 +1,5 @@
+import type { Location, Params } from "@remix-run/router";
 import type { ComponentType } from "react";
-import type { Location, Params } from "react-router-dom";
```

Comment:
`react-router-dom` isn't in `dependencies` anymore, so this would always be invalid

Probably a good idea to add an ESLint rule to catch these kind of things? 🤔

Reply:
yeah - I would have actually expected that to fail on the build but I guess the yarn workspace bites us there?  I feel like I recall this coming up in the past and maybe we never found a good solution then.  Do you know of one?  I've used https://www.npmjs.com/package/eslint-import-resolver-webpack before but that won't work here 😄

Reply:
ok yeah that feels like what we need, doesn't need to go so far as the webpack one above and actually resolve since we don't use aliases and such

Reply:
`eslint-plugin-import` should do the trick, specifically the `import/no-extraneous-dependencies` rule

Reply:
That's indeed the rule I was talking about!

I'll create a separate PR to enable it, so we're sure this never happens again